sweagent/environment/utils.py

```diff
@@ -420,12 +432,12 @@ def image_exists(image_name: str) -> bool:
         return False
     elif len(filterred_images) > 1:
         RuntimeError(f"Multiple images found for {image_name}, that's weird.")
-    attrs = filterred_images[0].attrs
-    if attrs is not None:
-        logger.info(
-            f"Found image {image_name} with tags: {attrs['RepoTags']}, created: {attrs['Created']} "
-            f"for {attrs['Os']} {attrs['Architecture']}.",
-        )
+    # attrs = filterred_images[0].attrs
```

Comment:
Don't log in side-effect-free functions.

Reply:
wait why not?

Reply:
We already have a log for `CACHED_IMAGE_NAME` that also logs the inputs that go into creating the hash in the name. Its more useful than this.